mappings/net/minecraft/inventory/Inventory.mapping

```diff
@@ -1,6 +1,6 @@
 CLASS agw net/minecraft/inventory/Inventory
 	METHOD K_ getSlotCapacity ()I
-	METHOD N_ getInventorySize ()I
+	METHOD N_ getTotalSlots ()I
```

Comment:
I feel `getSlotCount` is more reasonable (or just keep it as is!). `getTotalSlots` really doesn't seem that clear a name.

Reply:
I opted to not use `getSlotCount` because I felt it could be easily confused with `getSlotCapacity` (which returns the max count an item can be stacked to within slots).

Reply:
Yeah, I had that same problem with it. Honestly, just leave as is then - `getInventorySize` isn't perfect, but it's clearer than `getTotalSlots`.

Reply:
Resolved by 7af4c0e